BuildTools/Mintfile

```diff
@@ -1,2 +1,2 @@
-nicklockwood/SwiftFormat@0.49.6
-realm/SwiftLint@0.47.0
\ No newline at end of file
+nicklockwood/SwiftFormat
+realm/SwiftLint
```

Comment:
I'm not sure about this. While it's nice to keep everything updated, this is the only thing that keeps all our machines and CI running the same version - and eventually this could introduce breaking changes.

I'm curious though, were you running it through Mint?

Reply:
Agree with @peagasilva here — I think we should always pin version for the reason mentioned.

Reply:
Agree with @peagasilva here — I think we should always pin version for the reason mentioned.

Reply:
Also agree with @peagasilva . Isn't the issue more that we don't really want to run swiftlint and swift format for CI?

Reply:
Alright, I'll revert this change! @peagasilva my local SwiftFormat is newer than the script, so `mint` was installing an old version on every commit. Somehow, it didn't remember that install and it had to do it over and over again.. quite annoying! Let's see if we can fix that

Reply:
Were you using mint to run it? If not so, you should run it through it.

```sh
mint run -m {mintfile_path} SwiftFormat --config {.swiftformat_path} .
```